.github/workflows/workflows.yaml

```diff
@@ -336,6 +516,15 @@ jobs:
         run: |
           echo "VERSION=${{ steps.get_version.outputs.version }}" >> $GITHUB_ENV
 
+      - name: Get git tag version Debian arm64 # if needed
```

Comment:
I used another way to get the tag info because the method used in the existing steps fails in PRs - the word "merge" is then extracted and written into the file and folder names.

And since I couldn't test a complete release, I couldn't check, if the existing tag extraction method works, which would make `Get git tag version Debian arm64` redundant.

I hope, that was clear enough... Otherwise, you can text me via Telegram.

Reply:
I'll keep the Debian arm64 part for the version tag in action "Create release" for now. It works like this and if anyone can tell me the exact ouput of VERSION there and if it is some string like "1.0.9" without the "v", we can chang RELEASE_VERSION in line 592 into VERSION and remove the steps "Get git tag version Debian arm64" and "Set package version Debian arm64".

The comment is changed to "might be redundant".

Reply:
If we need a specific version tag for the deb, that is fine - but I would argue that comments like these unfortunately adds more confusion. I would argue that a good comment should state some fact, not ask a question - in this case I would say we just remove them actually.